network-api/networkapi/wagtailpages/templatetags/class_tags.py

```diff
@@ -0,0 +1,8 @@
+from django import template
+
+register = template.Library()
+
+
+@register.filter
```

Comment:
There's not much here, but I'm still not clear what it's doing when called on the template in the other file. Can you clarify?

Reply:
You can't call properties with underscores in their name in django templates. I don't know why, but that's the law. And since we need page object's `.__class__.__name__` the only way to do that is with a template filter.